docs/it/attributi_utente.rst

```diff
@@ -31,37 +28,28 @@ Tabella attributi utente
 
        Esempio: 
 
-       .. code-block::
-
-         "https://attributes.eid.gov.it/spid_cod":"ABCD123456789A"
+       ``"$PREFIX/spid_cod":"ABCD123456789A"``
```

Comment:
$PREFIX/spid_code

Reply:
@nunzionapoli please fix on https://www.agid.gov.it/sites/default/files/repository_files/tabella_attributi_v.1.3.pdf

Reply:
risolto